### PR TITLE
Raps fix

### PR DESCRIPTION
**Pull request by Collin-St**
I don't know why they changed the file paths for their CSVs, but they did. They also removed a field from the vaccine CSV.

### CODE REVIEW

File: scrapers/covid-19/getTherapeutics.js [hunk 1]
```diff
@@ -31,10 +31,10 @@ const getTherapeuticsData = async (keys, redis) => {
 		const date = html(`.small:first-of-type`).text().split(' ').slice(1, 4);
 		[day, month, year] = date;
 	} catch (err) {
-		logger.err('Error: Requesting therapeutics data failed!', err);
+		logger.err('Error: Requesting therapeutics data failed!456', err);
```
Comment (ebwinters) [inline scrapers/covid-19/getTherapeutics.js:34]: What is 456

File: scrapers/covid-19/getTherapeutics.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ const getTherapeuticsData = async (keys, redis) => {
 			data: cleanData(parsedData)
 		}));
 	} catch (err) {
-		logger.err('Error: Requesting therapeutics data failed!', err);
+		logger.err('Error: Requesting therapeutics data failed!123', err);
```
Comment (ebwinters) [inline scrapers/covid-19/getTherapeutics.js:46]: Oh I see, let's just change the message instead of adding numbers

**Review by ebwinters — COMMENTED** [review]
Let's logs to be English instead of numbers then it's good to go

File: scrapers/covid-19/getTherapeutics.js (hunk 2 above)
Comment (Collin-St) [reply scrapers/covid-19/getTherapeutics.js:46]: Oh crap, let me change that

**Review by ebwinters — APPROVED** [review]

**Pull request merged**